base/shell/explorer/lang/tr-TR.rc

```diff
@@ -139,7 +140,7 @@ BEGIN
     PUSHBUTTON       "&Ekle...", IDC_CLASSICSTART_ADD, 162, 18, 53, 14
     PUSHBUTTON       "&Sil...", IDC_CLASSICSTART_REMOVE, 162, 33, 53, 14
     PUSHBUTTON       "&Gelişmiş", IDC_CLASSICSTART_ADVANCED, 162, 49, 53, 14
-    PUSHBUTTON       "&Diz", IDC_CLASSICSTART_SORT, 162, 65, 53, 14
+    PUSHBUTTON       "S&ırala", IDC_CLASSICSTART_SORT, 162, 65, 53, 14
```

Comment:
If `ı` is the same letter on the turkish keyboard as `i` then
`PUSHBUTTON       "S&ırala", IDC_CLASSICSTART_SORT, 162, 65, 53, 14`
will collide with
`PUSHBUTTON       "S&il", IDC_CLASSICSTART_CLEAR, 162, 94, 53, 14`

So unless you do have a very good reason for this change (that I would lack to see myself), then I would recommend to revert that single-line-change.
The rest of your PR looks ok to me.

Reply:
the reason I wrote sort was for better understanding & where can I place the location? both are really important

Reply:
> If `ı` is the same letter on the turkish keyboard as `i` then
> `PUSHBUTTON       "S&ırala", IDC_CLASSICSTART_SORT, 162, 65, 53, 14`
> will collide with
> `PUSHBUTTON       "S&il", IDC_CLASSICSTART_CLEAR, 162, 94, 53, 14`
> 
> So unless you do have a very good reason for this change (that I would lack to see myself), then I would recommend to revert that single-line-change.
> The rest of your PR looks ok to me.

Not actually. What are u talking about is for English keyboards. Here is a Turkish Q keyboard. ![IMG_20240903_190656.jpg](https://github.com/user-attachments/assets/89edd7c5-628e-4ed8-87ea-1b34a4f81412)

'ı' and 'i' are different.

Reply:
Thank you for that elaboration. Resolved then.

Reply:
> If `ı` is the same letter on the turkish keyboard as `i` then `PUSHBUTTON "S&ırala", IDC_CLASSICSTART_SORT, 162, 65, 53, 14` will collide with `PUSHBUTTON "S&il", IDC_CLASSICSTART_CLEAR, 162, 94, 53, 14`
> 
> So unless you do have a very good reason for this change (that I would lack to see myself), then I would recommend to revert that single-line-change. The rest of your PR looks ok to me.

![image](https://github.com/user-attachments/assets/f419068f-8e00-4287-80d2-091e59725c10)
nope bro its working :D